js-miniapp-sample/src/pages/uri-schemes.js

```diff
@@ -24,14 +29,41 @@ const useStyles = makeStyles((theme) => ({
     fontWeight: 'bold',
     paddingBottom: 0,
   },
-  actions: {
-    justifyContent: 'center',
-    paddingBottom: 16,
+  textfield: {
+    width: '100%',
   },
 }));
 
 const UriSchemes = () => {
+  const EXTERNAL_WEBVIEW_URL =
+    'https://htmlpreview.github.io/?https://raw.githubusercontent.com/rakutentech/js-miniapp/master/js-miniapp-sample/external-webview/index.html';
   const classes = useStyles();
+  const [params, setParams] = useState('?testSendParam=someValue&test2=test2');
+
+  function validateParams(params) {
+    if (!params.startsWith('?') || params.indexOf('=') <= -1) {
+      return false;
+    }
+
+    return true;
+  }
+
+  function onOpenExternalWebview() {
+    if (params && !validateParams(params)) {
+      window.alert(
+        'Invalid params. Please input params in the format ?param1=value1&param2=value2'
+      );
+      return;
+    }
+
+    const callbackUrl = `${window.location.protocol}//${window.location.host}/index.html`;
```

Comment:
It is more ideal to redirect to `URI Schemes` page
but this redirect to landing page is okay if we don't want to spend much effort on sample app 😄

Reply:
Yeah, that's true - this is much simpler to do 😆 

Also, we can re-use this for showing params that are passed to the mini app when created.